packages/nx-plugin/package.json

```diff
@@ -3,9 +3,18 @@
   "version": "0.29.0",
   "license": "MIT",
   "description": "Nx plugin to integrate the Code PushUp CLI into your workspace 🛠️",
+  "homepage": "code-pushup.dev",
+  "bugs": {
+    "url": "https://github.com/code-pushup/cli/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/code-pushup/cli.git",
+    "directory": "packages/nx-plugin"
+  },
```

Comment:
ATM the Nx plugin does not use esbuild, so we have to maintain that part manually.
In the future we should split `package.json` automation from `esbuild` plugin.